src/axom/slic/docs/sphinx/sections/citations.rst

```diff
@@ -5,10 +5,10 @@
 
 .. _LLNL: https://www.llnl.gov/
 .. _Shroud: https://shroud.readthedocs.io/en/develop/
-.. _Lumberjack: https://lc.llnl.gov/axom/docs/sphinx/lumberjack_docs/html/index.html
-.. _Axom Quick Start Guide: https://axom.readthedocs.io/en/develop/docs/sphinx/quickstart_guide/index.html
-.. _Axom Toolkit: https://axom.readthedocs.io/en/develop/
+.. _Lumberjack: ../../../../lumberjack/docs/sphinx/index.html
```

Comment:
Sphinx complains about this line and the next two.  I suspect this file needs to be removed.  See also mint's citations.rst.

Reply:
Sorry I didn't see this one was included, as is mint's citiation.rst.  Good reason not to delete it.